pkg/sdk/go/sdk.go

```diff
@@ -1001,21 +1018,24 @@ func NewSDK(conf Config) SDK {
 
 // processRequest creates and send a new HTTP request, and checks for errors in the HTTP response.
 // It then returns the response headers, the response body, and the associated error(s) (if any).
-func (sdk mfSDK) processRequest(method, url, token, contentType string, data []byte, expectedRespCodes ...int) (http.Header, []byte, errors.SDKError) {
+func (sdk mfSDK) processRequest(method, url, token string, data []byte, requestHeaders map[string]string, expectedRespCodes ...int) (http.Header, []byte, errors.SDKError) {
```

Comment:
```suggestion
func (sdk mfSDK) processRequest(method, url, token, contentType string, data []byte, headers map[string]string, expectedRespCodes ...int) (http.Header, []byte, errors.SDKError) {
```
I thought this would be the new function signature

Reply:
There is no need to pass `contentType` as a separate argument since its also a header